package.json

```diff
@@ -1,13 +1,11 @@
 {
   "name": "daffodil-debugger",
-  "displayName": "Daffodil Debugger",
+  "displayName": "Apache Daffodil Debugger Extension",
+  "description": "VS Code extension for Apache Daffodil Schema Debugger",
```

Comment:
Seems `VSCode` without the space is the common way to refer to Visual Studio Code? I would also refer it as ` `DFDL Schema Debugger`, just to make it clear that these are Apache Daffodil schemas, they are DFDL Schemas, and the ebugger is powered by Apache Daffodil.

Reply:
> Seems VSCode without the space is the common way to refer to Visual Studio Code

You see it both ways in the wild.  I was looking at Openwhisk json when updating this.  Want it changed?

>  would also refer it as DFDL Schema Debugger`

OK

Reply:
You're right, the first couple things I saw were VSCode, but I see both uses, even in official vscode documentation. This is fine.

Reply:
Yeah I just did a search and we have a mix in this repo, but definitely more with the space.

I added #24 to get this project to a standardization.